OneCommunityGlobal/HighestGoodNetworkApp
# Lucas revert to time log

# Description
(PRIORITY LOW) Rafael/Jae: Fix Tasks revert to Time Log tab for users OTHER THAN VOLUNTEER 
Only applies to Volunteer accounts, since all other account roles should always stay with the Task tab as default.  (WIP Lucas)
Tested [THIS PR 734](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/734) and accounts other than Volunteer are now reverting to Timelog tab too. Not all, but some. Need a new PR that fixes this so ONLY Volunteer accounts display this function. 
If a VOLUNTEER has no tasks, then their default tab should be "Current Week Timelog" not "Tasks". If their tasks are resolved via the green checkmark, and they have none left, then their view should return to "Current Week Timelog".

## Main changes explained:
Once a volunteer user has no task, the Time Log tab should be the default tab for that user

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. give task to volunteer user
5. check if the Tasks Tab is shown first for that user
6. dismiss the task
7. refresh the volunteer user page and check if the Timelog tab is activated by default

### HOW TO CREATE A TASK, ASSIGN IT TO AN USER AND TEST MY PR


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92558269/40a95abc-aaab-4e4c-bbf0-060aaeac1186
